src/test/java/com/epam/repository/ReservationDAOTest.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.repository;
+
+import com.epam.model.Reservation;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.when;
+
+public class ReservationDAOTest {
+    @InjectMocks
+    ReservationDAO reservationDAO;
+    @Mock
+    private ResultSet resultSet;
+    @Mock
+    private Reservation expectedReservation;
+
+    private Integer testClientId = 1;
+    private Integer testTourOfferId = 1;
+    private Integer testNumberOfPeople = 1;
+    private String testStatus = "paid";
+    private Integer testDiscountId = 1;
+    private Integer testTotalPrice = 10_000;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void buildPersonPositiveCheck() throws SQLException {
```

Comment:
See the @SneakyThrows annotation for the exceptions

Reply:
done